san-francisco-tourism-solution/src/test/java/com/javaone/hol2017/junit5/EarthquakeTest.java

```diff
@@ -1,12 +1,12 @@
 package com.javaone.hol2017.junit5;
 
-import static org.junit.jupiter.api.Assertions.*;
-import static org.hamcrest.Matchers.*;
-import static org.hamcrest.MatcherAssert.*;
+import com.javaone.hol2017.junit5.Earthquake.ShakeException;
```

Comment:
Can you revert these changes and make the code still work?
less is more in a Pull request 👍